ohsome/helper.py

```diff
@@ -204,29 +204,34 @@ def format_bboxes(
 
 
 def format_bpolys(
-    bpolys: Union[gpd.GeoDataFrame, shapely.Polygon, shapely.MultiPolygon, str]
+    bpolys: Union[
+        gpd.GeoDataFrame, gpd.GeoSeries, shapely.Polygon, shapely.MultiPolygon, str
+    ]
 ) -> str:
     """
     Formats bpolys parameter to comply with ohsome API
     :param
     bpolys: Polygons given as geopandas.GeoDataFrame or string formatted as GeoJSON FeatureCollection.
     :return:
     """
-    if isinstance(bpolys, gpd.GeoDataFrame) or isinstance(bpolys, gpd.GeoSeries):
-        return bpolys.to_json(na="drop")
+    if isinstance(bpolys, gpd.GeoDataFrame):
+        return bpolys.to_json(na="drop", show_bbox=False, drop_id=False, to_wgs84=True)
+    elif isinstance(bpolys, gpd.GeoSeries):
+        return format_bpolys(bpolys.to_frame("geometry"))
     elif isinstance(bpolys, shapely.Polygon) or isinstance(
         bpolys, shapely.MultiPolygon
     ):
-        return format_bpolys(gpd.GeoDataFrame(geometry=[bpolys]))
+        return format_bpolys(gpd.GeoDataFrame(geometry=[bpolys], crs="EPSG:4326"))
```

Comment:
Is `shapely.to_geojson` more performant then converting to geodataframe and then to json?

Reply:
`shapely.to_geojson` will only create a geojson geometry but we need a feature collection as of the definition of the Ohsome API. The performance gain should be negligible.